src/hooks/queries/gift/usePostGift.tsx

```diff
@@ -12,6 +12,8 @@ export const usePostGift = (
   roomId: number,
   targetDate: string,
   setStep: React.Dispatch<React.SetStateAction<number>>,
+  updateAddGiftInfo: (newInfo: Partial<AddGiftInfo>) => void,
```

Comment:
추후에는 props로 받는 방법 이외에,

```suggestion

 const { updateAddGiftInfo } = use뭐시기Context;
 
  이렇게 불러와서
  
  updateAddGiftInfo({ name: '', cost: 0, imageUrl: '', url: '' });
  
  바로 사용해도 좋을 것 같습니다! 아무래도 props type까지 설정해줘야하고 살짝 귀찮을 수 있으니 :)
  
```

Reply:
피드백 감사합니다~! 큐에이 하면서도 저 부분 때문에 귀찮아지더라고요ㅠ 그래서 giftee name 관련 context를 추가할 때는 제안해주신 방식과 같은 방식으로 직접 불러오는 방식으로 구현했습니다. 이 부분은 나중에 싹 리팩토링 하겠습니다~!